Example/Pods/Pods.xcodeproj/project.xcworkspace/contents.xcworkspacedata

```diff
@@ -1,4 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Workspace
    version = "1.0">
+   <FileRef
+      location = "self:">
+   </FileRef>
```

Comment:
I'm not sure what this file is. Might need to be added to the git ignore. Was this a conscientious change?

Reply:
This wasn't a conscientious change, not sure what caused that.